src/style.css

```diff
@@ -2,0 +1,73 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+* {
+  font-family: Avenir-Book, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  color: rgba(61, 78, 97, 1);
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
```

Comment:
You have a margin-top on the .search-bar class. If there are no specific requirements for the container size, consider letting the content size dictate the spacing instead of fixing it with margins.